<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_PrintPreviewOptions</class>
 <widget class="QWidget" name="QG_PrintPreviewOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Print Preview Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QComboBox" name="cbScale">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="bBlackWhite">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Toggle Black / White mode</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="png/icons.qrc">
       <normaloff>:/ui/icons/printbw.png</normaloff>:/ui/icons/printbw.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="bCenter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Center to page</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="png/icons.qrc">
       <normaloff>:/ui/icons/printcenter.png</normaloff>:/ui/icons/printcenter.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="bFit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Fit to page</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="png/icons.qrc">
       <normaloff>:/ui/icons/printfit.png</normaloff>:/ui/icons/printfit.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="sep1_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">rs_line.h</include>
  <include location="local">rs_settings.h</include>
  <include location="local">rs_modification.h</include>
  <include location="local">rs_actionprintpreview.h</include>
 </includes>
 <resources>
  <include location="png/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bCenter</sender>
   <signal>clicked()</signal>
   <receiver>QG_PrintPreviewOptions</receiver>
   <slot>center()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBlackWhite</sender>
   <signal>toggled(bool)</signal>
   <receiver>QG_PrintPreviewOptions</receiver>
   <slot>setBlackWhite(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScale</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_PrintPreviewOptions</receiver>
   <slot>scale(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScale</sender>
   <signal>activated(QString)</signal>
   <receiver>QG_PrintPreviewOptions</receiver>
   <slot>scale(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bFit</sender>
   <signal>clicked()</signal>
   <receiver>QG_PrintPreviewOptions</receiver>
   <slot>fit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
